api-participant/src/main/java/bio/terra/pearl/api/participant/service/AuthUtilService.java

```diff
@@ -38,11 +38,15 @@ public AuthUtilService(
    * returns the enrollee if the user is authorized to access/modify it, throws an error otherwise
    */
   public Enrollee authParticipantUserToEnrollee(UUID participantUserId, String enrolleeShortcode) {
-    // for now, a user is only allowed to access an enrollee if it's themself.  Later, we'll add
-    // proxies
+    // for now, a user is only allowed to access an enrollee if it's themselves or their proxy
     Optional<Enrollee> enrolleeOpt =
         enrolleeService.findByParticipantUserIdAndShortcode(participantUserId, enrolleeShortcode);
     if (enrolleeOpt.isEmpty()) {
+      Enrollee targetEnrollee =
+          enrolleeRelationService.isUserProxyForEnrollee(participantUserId, enrolleeShortcode);
+      if (targetEnrollee != null) {
+        return targetEnrollee;
+      }
```

Comment:
continuing our trend of avoid `opt` variables, I think this gets cleaner if you refactor the enrolleeRelationService method to return an optional.
```suggestion
    Enrollee enrollee =
        enrolleeService.findByParticipantUserIdAndShortcode(participantUserId, enrolleeShortcode)
          orElseGet(() -> enrolleeRelationService.getEnrolleeByProxyUserId(participantUserId, enrolleeShortcode).orElseThrow(() -> new PermissionDeniedException(...));

```